prereise/gather/demanddata/transportation_electrification/mileage.py

```diff
@@ -0,0 +1,140 @@
+    :return: (*np.ndarray*) array of 1s and 2s indicating weekend/weekday designations for the model year.
+
+    """
+    return model_year_dti.dayofweek.isin(range(5)).astype(int) + 1
+
+
+def get_input_month(model_year_dti: pd.DatetimeIndex) -> np.ndarray(int):
+    """Determine month of each day
+
+    :param pd.DatetimeIndex model_year_dti: a DatetimeIndex encompassing the model year.
+    :return: (*np.ndarray*) -- stores the month the day is in for each day of the year.
+    """
+    return model_year_dti.month
+
+
+def get_data_month(data: pd.DataFrame) -> pd.Series:
+    """Get month value from data.
+
+    :param pd.DataFrame data: the data to get months from.
+    :return: (*pd.Series*) -- list of months.
+    """
+    return data["Date"].dt.month
+
+
+def get_data_day(data: pd.DataFrame) -> np.array(int):
+    """Get weekday/weekend designation value from data.
+
+    :param pd.DataFrame data: the data to get day of week from.
+    :return: (*np.array*) -- indicates weekend or weekday for every day.
+    """
+    return data["If Weekend"]
+
+
+def load_data(census_region: int, filepath: str = "nhts_census.mat") -> pd.DataFrame:
+    """Load the data at nhts_census.mat.
+
+    :param int census_region: the census region to load data from.
+    :param str filepath: the path to the matfile.
+    :raises ValueError: if the census division is not between 1 and 9, inclusive.
+    :return: (*pd.DataFrame*) -- the data loaded from nths_census.mat, with column
+        names added.
+    """
+    if not (1 <= census_region <= 9):
+        raise ValueError("census_region must be between 1 and 9 (inclusive).")
+
+    # Somehow load in the matfile
+    nhts_census = loadmat(filepath)
+    # Load the data for the desired region
+    raw_data = nhts_census[f"census_{census_region}_sorted"]
+    # Convert to data frame, adding column names
+    df = pd.DataFrame(raw_data, columns=const.nhts_census_column_names)
+
+    return df
+
+
+def remove_ldt(data: pd.DataFrame) -> pd.DataFrame:
+    """Remove light duty trucks from data loaded from nths_census.mat.
+
+    :param pd.DataFrame data: the data returned from load_data.
+    :return: (*pd.DataFrame*) -- the data loaded from load_data with all rows
+        involving LDT removed.
+    """
+
+    return data.loc[data["Vehicle type"] < 5].copy()
+
+
+# ldt = light duty truck
+# ldv = light duty vehicle
+def total_daily_vmt(
+    census_region: int,
+    comm_type: int,
+    locationstrategy: list[int],
+    input_day: list[int],
+) -> np.array[float]:
+    """load data and use the parameters to calculate total_daily_vmt.
+
+    :param int census_region: the type of census
+    :param int comm_type: the type of Commute
+    :param list locationstrategy: strategy for each location
+    :param list input_day: day of the week for each day in the year derived from
+        first_func
+    :return: (*list*) -- daily_vmt_total each row is a year of entries for each vehicle

```

Comment:
list --> np.array for the returned variable type. Also, the name (`daily_vmt_total`) of the should not be mentioned in the description. For an array I would use element in place of row